model/network/registry.yaml

```diff
@@ -236,3 +236,46 @@ groups:
         type: string
         brief: 'The network interface name.'
         examples: [ 'lo', 'eth0' ]
+      - id: network.connection.state
+        type:
+          members:
+            - id: close
+              value: 'close'
+              stability: experimental
+            - id: close_wait
+              value: 'close_wait'
+              stability: experimental
+            - id: closing
+              value: 'closing'
+              stability: experimental
+            - id: delete
```

Comment:
I don't think there is a delete state in the RFC - do we need to add it?

Reply:
can we postpone providing it? It's not in the RFC and it's always easy to extend enum in semconv.

Reply:
Removed that.